source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -455,6 +456,107 @@ message TransformationAccessChain {
+  // |                                             | %26 = OpBitFieldUExtract %2 %6 %11 %6 ; extracts bit 3 from %6 |
+  // |                                             | %27 = OpBitwiseOr %2 %25 %26                                   |
+  // |                                             |                                                                |
+  // |                                             | %28 = OpBitFieldUExtract %2 %5 %12 %6 ; extracts bit 4 from %5 |
+  // |                                             | %29 = OpBitFieldUExtract %2 %6 %12 %6 ; extracts bit 4 from %6 |
+  // |                                             | %30 = OpBitwiseOr %2 %28 %29                                   |
+  // |                                             |                                                                |
+  // |                                             | %31 = OpBitFieldUExtract %2 %5 %13 %6 ; extracts bit 5 from %5 |
+  // |                                             | %32 = OpBitFieldUExtract %2 %6 %13 %6 ; extracts bit 5 from %6 |
+  // |                                             | %33 = OpBitwiseOr %2 %31 %32                                   |
+  // |                                             |                                                                |
+  // |                                             | %34 = OpBitFieldUExtract %2 %5 %14 %6 ; extracts bit 6 from %5 |
+  // |                                             | %35 = OpBitFieldUExtract %2 %6 %14 %6 ; extracts bit 6 from %6 |
+  // |                                             | %36 = OpBitwiseOr %2 %34 %35                                   |
+  // |                                             |                                                                |
+  // |                                             | %37 = OpBitFieldUExtract %2 %5 %15 %6 ; extracts bit 7 from %5 |
+  // |                                             | %38 = OpBitFieldUExtract %2 %6 %15 %6 ; extracts bit 7 from %6 |
+  // |                                             | %39 = OpBitwiseOr %2 %37 %38                                   |
+  // |                                             |                                                                |
+  // |                                             | %40 = OpBitFieldInsert %2 %18 %21 %6 %6 ; inserts bit 1        |
+  // |                                             | %41 = OpBitFieldInsert %2 %40 %24 %10 %6 ; inserts bit 2       |
+  // |                                             | %42 = OpBitFieldInsert %2 %41 %27 %11 %6 ; inserts bit 3       |
+  // |                                             | %43 = OpBitFieldInsert %2 %42 %30 %12 %6 ; inserts bit 4       |
+  // |                                             | %44 = OpBitFieldInsert %2 %43 %33 %13 %6 ; inserts bit 5       |
+  // |                                             | %45 = OpBitFieldInsert %2 %44 %36 %14 %6 ; inserts bit 6       |
```

Comment:
You could also replace this with a `...`.

Reply:
An alternative (or you could do it as well) would be to move the example to a comment in the header file of the transformation class, and remark here that the reader should see there for an example.